<!-- -*- mode: XML -*- -->
<launch>
    <node pkg="odrive_ros" type="odrive_node" name="odrive" output="screen">
        <param name="wheelbase" value="0.3"/>
        <param name="tyre_circumference" value="0.35"/>
        <param name="encoder_counts_per_rev" value="4096"/>
    </node>
</launch>
